trace_decoder/src/decoding.rs

```diff
@@ -242,9 +242,9 @@ impl ProcessedBlockTrace {
         txn_idx: TxnIdx,
     ) {
         let txn_k = Nibbles::from_bytes_be(&rlp::encode(&txn_idx)).unwrap();
-        trie_state.txn.insert(txn_k, meta.txn_bytes());
+        let _ = trie_state.txn.insert(txn_k, meta.txn_bytes());
```

Comment:
why not using `unwrap` as the other methods? (to have consistent behavior, i.e. panic upon failure).

Reply:
Or just handle the errors properly in both cases

Reply:
I'll create an issue to add error handling to the `mpt_trie` calls in `decoding` since #118 landed.